tajo-plan/src/main/java/org/apache/tajo/plan/rewrite/rules/FilterPushDownRule.java

```diff
@@ -358,7 +353,21 @@ public LogicalNode visitJoin(FilterPushDownContext context, LogicalPlan plan, Lo
       context.pushingDownFilters.removeAll(matched);
     }
 
-    context.pushingDownFilters.addAll(outerJoinFilterEvalsExcludePredication);
+    matched.clear();
+    matched.addAll(outerJoinFilterEvalsExcludePredication);
```

Comment:
I'll change those variable names in https://issues.apache.org/jira/browse/TAJO-1350.